lte/gateway/python/magma/pipelined/app/enforcement_stats.py

```diff
@@ -472,13 +468,13 @@ def _handle_flow_stats(self, stats_msgs):
             return
         # Send report even if usage is empty. Sessiond uses empty reports to
         # recognize when flows have ended
-        self._report_usage(current_usage)
+        self._report_usage(current_usage[1])
 
         # This is done primarily for CWF integration tests, TODO rm
         self.total_usage = current_usage
         # Report only if their is no change in version
         if self.ng_config.ng_service_enabled == True:
-            self._prepare_session_config_report(stats_msgs)
+            self._prepare_ruleRecord_report(current_usage[0])
```

Comment:
why are you taking only the first element

Reply:
I explained in above. please suggest if will have better approach.